lib/phase/flatten.js

```diff
@@ -3,6 +3,37 @@
 const Hoek = require('hoek');
 const clone = require('clone');
 
+/**
+ * Check if a job has duplicate steps
+ * @method checkAdditionalRules
+ * @param  {Object}       parsedDoc Document that went through structural parsing
```

Comment:
`parsedDoc` => `doc`?
Or is `jobs` enough for this function?

Reply:
Either way is fine, but I focused on scalability.